extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -33,6 +33,8 @@ function ($, mw, loader, nirvana, tracker, labeling) {
 		typeGetter = typeGetterProvided;
 		$typeLabel = $('.template-classification-type-text');
 
+		$(w).bind('keydown', openModalKeyboardShortcut);
```

Comment:
What about unbinding for a time that modal is open or at least double opening prevent.